lib/ui_pages/about_page.dart

```diff
@@ -8,6 +9,11 @@ class AboutPage extends StatelessWidget {
     return Scaffold(
       endDrawer: CustomDrawer(),
       appBar: AppBar(
+        leading: IconButton(
+          icon: Icon(Icons.arrow_back),
+          onPressed: () =>
+              Navigator.pushReplacementNamed(context, RouteName.homePage),
```

Comment:
ovdje neznate hoce li doci u about iz home pagea.

Defaultni leading je bio dobar. samo <- da napravi pop.
To treba ostati isto